src/components/search/SearchApp.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+
+// component imports
+import SearchForm from "./SearchForm";
+import SearchResults from "./SearchResults";
+const queryClient = new QueryClient();
```

Comment:
Does this need to be inside `SearchApp`? I haven't used `react-query` in a while

Reply:
What do you mean by "this"? the query client? or the component imports? or both? 

I believe you have to wrap any components you want to access your useQuery hook with the queryClient provider?

Reply:
Oops, I meant this: 

```ts
const queryClient = new QueryClient();
```

But I just checked the docs and see the example here: https://tanstack.com/query/v4/docs/react/adapters/react-query#example

TIL! Disregard this!